website/docs/r/appstream_fleet.html.markdown

```diff
@@ -0,0 +1,83 @@
+---
+subcategory: "AppStream"
+layout: "aws"
+page_title: "AWS: aws_appstream_fleet"
+description: |-
+Provides an AppStream fleet
+---
+
+# Resource: aws_appstream_fleet
+
+Provides an AppStream fleet.
+
+## Example Usage
+
+```hcl
+resource "aws_appstream_fleet" "test_fleet" {
+  name = "test-fleet"
+  compute_capacity {
+    desired_instances = 1
+  }
+  description                        = "test fleet"
+  idle_disconnect_timeout_in_seconds = 15
+  display_name                       = "test-fleet"
+  enable_default_internet_access     = false
+  fleet_type                         = "ON_DEMAND"
+  image_name                         = "Amazon-AppStream2-Sample-Image-02-04-2019"
+  instance_type                      = "stream.standard.large"
+  max_user_duration_in_seconds       = 600
+  vpc_config {
+    subnet_ids = ["subnet-06e9b13400c225127"]
+  }
+  tags = {
+    TagName = "tag-value"
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) A unique name for the fleet.
+* `name_prefix` -  (Optional) Creates a unique name beginning with the specified prefix. Conflicts with `name`.
+* `compute_capacity` - (Required) The desired capacity for the fleet.
+  * `desired_instances` - (Required) The desired number of streaming instances.
```

Comment:
misaligned

Reply:
`desired_instances` it's a paremeter from the `compute_capacity` object